components/folders/AdditionalInfoCard/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import DeputyPreview from "../DeputyPreview";
+import Typography from "@mui/material/Typography";
+import Stack from "@mui/material/Stack";
+import useTheme from "@mui/material/styles/useTheme";
+
+const AdditionalInfoCard = () => {
+  const theme = useTheme();
+  return (
+    <Card elevation={0} sx={{ backgroundColor: theme.palette.grey[50] }}>
```

Comment:
Some props are theme aware to avoid having to use the `useTheme` hook every where
```suggestion
    <Card elevation={0} sx={{ backgroundColor: "grey.50" }}>
```